src/utils/ui/theme/styles.css

```
@@ -0,0 +1,376 @@
+
+  /* Destructive: red alert */
+  --destructive: oklch(0.55 0.2 25);
+  --destructive-foreground: oklch(0.95 0 0);
+
+  /* Borders with amber glow */
+  --border: oklch(0.35 0.05 75);
+  --input: oklch(0.2 0.03 280);
+  --ring: oklch(0.75 0.18 75);
+}
+
+@theme inline {
```

Stylelint (17.3.0)

[error] Unexpected unknown at-rule "@theme" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)